bin/runFixedPop

```diff
@@ -216,18 +216,24 @@ if __name__ == '__main__':
 
         # Select the initial binary sample method from user input
         if args.initial_samp == 'independent':
-            IBT, sampled_mass, n_samp = InitialBinaryTable.sampler(args.initial_samp, kstar1_range, kstar2_range, 'kroupa93', 'thermal', SFH_model, component_age, args.metallicity, args.Nstep)
+            IBT, mass_singles, mass_binaries, n_singles, n_binaries = InitialBinaryTable.sampler(args.initial_samp, kstar1_range, kstar2_range, 0.5, 'kroupa93', 'thermal', SFH_model, component_age, args.metallicity, args.Nstep)
 
         if args.initial_samp == 'multidim':
-            IBT, sampled_mass, n_samp = InitialBinaryTable.sampler(args.initial_samp, kstar1_range, kstar2_range, rand_seed, args.nproc, SFH_model, component_age, args.metallicity, args.Nstep)
+            IBT, mass_singles, mass_binaries, n_singles, n_binaries= InitialBinaryTable.sampler(args.initial_samp, kstar1_range, kstar2_range, rand_seed, args.nproc, SFH_model, component_age, args.metallicity, args.Nstep)
 
         # Log the total sampled mass from the initial binary sample
         # for future Galactic occurence rate calculation
-        total_sampled_mass += sampled_mass
+        total_mass_singles += mass_singles
+        total_mass_binaries += mass_binaries
+        total_n_singles += n_singles
+        total_n_binaries += n_binaries
```

Comment:
I don't think these lines will work, since you never initialized the variables `total_mass_singles`, `total_mass_binaries`, `total_n_singles`, and `total_n_binaries`. Shouldn't you do this at the try/except statement at 169?